warehouse/models/mart/transit_database/bridge_schedule_dataset_for_validation.sql

```diff
@@ -16,7 +16,7 @@ bridge_schedule_dataset_for_validation AS (
         LEAST(datasets._valid_to, sched_ref._valid_to) AS _valid_to
     FROM datasets
     INNER JOIN datasets AS sched_ref
-        ON datasets.schedule_to_use_for_rt_validation_gtfs_dataset_key = sched_ref.original_record_id
+        ON datasets.schedule_to_use_for_rt_validation_gtfs_dataset_key = sched_ref.source_record_id
```

Comment:
this reads weirdly but `schedule_to_use_for_rt_validation_gtfs_dataset_key` is just an aliased column

Reply:
Yes we should probably rename it, I got lazy with key vs ID at some point. But I don't think it's worth doing here